src/sys_uv.zig

```diff
@@ -350,13 +350,13 @@ pub fn pwritev(fd: FileDescriptor, bufs: []const bun.PlatformIOVec, position: i6
         for (bufs) |buf| {
             total_bytes += buf.len;
         }
-        log("uv write({}, {d} total bytes) = {d} ({any})", .{ uv_fd, total_bytes, rc.value, debug_timer });
+        log("uv write({}, {d} total bytes) = {d} ({any})", .{ uv_fd, total_bytes, rc.int(), debug_timer });
     }
 
     if (rc.errno()) |errno| {
         return .{ .err = .{ .errno = errno, .fd = fd, .syscall = .write, .from_libuv = true } };
     } else {
-        return .{ .result = @as(usize, @intCast(rc.value)) };
+        return .{ .result = @as(usize, @intCast(rc.int())) };
```

Comment:
What's the `.value` to `.int()` change about?

Reply:
`ReturnCode` was a struct and now is a enum, so we dont have a `.value` anymore